storage/src/adb/any/variable/mod.rs

```diff
@@ -217,20 +217,8 @@ impl<E: RStorage + Clock + Metrics, K: Array, V: Codec, H: CHasher, T: Translato
     async fn build_snapshot_from_log(mut self) -> Result<Self, Error> {
         // Align the mmr with the location map. Any elements we remove here that are still in the
         // log will be re-added later.
-        let mut mmr_leaves = leaf_pos_to_num(self.mmr.size()).unwrap();
-        let locations_size = self.locations.size().await?;
-        if locations_size > mmr_leaves {
-            warn!(
-                mmr_leaves,
-                locations_size, "rewinding misaligned locations map"
-            );
-            self.locations.rewind(mmr_leaves).await?;
-            self.locations.sync().await?;
-        } else if mmr_leaves > locations_size {
-            warn!(mmr_leaves, locations_size, "rewinding misaligned mmr");
-            self.mmr.pop((mmr_leaves - locations_size) as usize).await?;
-            mmr_leaves = locations_size;
-        }
+        let mut mmr_leaves =
+            super::super::align_mmr_and_locations(&mut self.mmr, &mut self.locations).await?;
```

Comment:
this `super::super` is not ideal. I suggest just importing `align_mmr_and_locations`

Reply:
fixed